packages/output/src/browser/output-widget.tsx

```diff
@@ -88,7 +88,27 @@ export class OutputWidget extends ReactWidget {
     }
 
     protected render(): React.ReactNode {
-        return <React.Fragment>{this.renderChannelSelector()}{this.renderChannelContents()}</React.Fragment>;
+        return <React.Fragment>
+            <div id='outputOverlay'>
+                {this.renderChannelSelector()}
+                {this.renderClearButton()}
+            </div>
+            {this.renderChannelContents()}
+        </React.Fragment>;
+    }
+
+    private readonly OUTPUT_CLEAR_BUTTON_ID = 'outputClear';
```

Comment:
Let's have a `const` somewhere for the CSS classes, or use `static` fields instead. Can you please update the other classes too? We [did this](https://github.com/theia-ide/theia/blob/e619393d725bbf5a05b260169193c7c538e55e4c/packages/git/src/browser/git-widget.tsx#L653-L669) in other widgets.

Reply:
This is an id, not a css class, but I can certainly extract it in the same fashion.